### ISSUE TITLE
[request] Options regarding tooltips

### ISSUE THREAD

**Issue by DarthWound** [micheleg/dash-to-dock#467]
Hi, thank you for this great extension!

It would be nice to have several options for tooltips (when hovering icons):
- hide/disable
- customize appearance

**Comment by corebots** [comment]
i second this. me too I'm looking for a way to make the tooltips (app names) display instantly.

**Comment by micheleg** [comment]
I'm not sure I want to enter into the appearance of the tooltips. I am instead considering adding the windows previews on mouse hover. In such case an option to disable any tooltip could be trivially included.

@corebots aren't tooltips already displayed instantly?

**Comment by DarthWound** [comment]
Well I understand :) But an option to disable them would be great.

As a workaround I customize their appearance in a custom gnome-shell.css in _~/.themes_.
Here are my tweaks, if it helps anybody:

```
.dash-label {
	font-size: 9pt;
  	border-radius: 11px;
	border: 1px solid #B5B7B9;
  	padding: 3px 8px;
  	color: #D5D7D9;
  	background-color: #050709;
}
```

**Comment by micheleg** [comment]
Yes, I'd like to leave the appearance customization to the themes. As a general idea, I tried to provide those changes that might be important given that the dock is shown on the desktop rather than only in the overview for which the themes might be designed (e.g. transparency)

**Comment by DarthWound** [comment]
No problem :) I close this request but feel free to re-open it if needed.

**Comment by micheleg** [comment]
I keep it open as a reminder.

**Comment by Scotty-Trees** [comment]
Wait, how can I disable tooltips because I don't like to see them? Btw, thanks very much for an amazing extension and all the work you do.

**Comment by DarthWound** [comment]
> Wait, how can I disable tooltips because I don't like to see them? Btw, thanks very much for an amazing extension and all the work you do.

AFAIK the only way would be creating a custom theme to hide them, it's a dirty hack I agree. Maybe there's an extension to really disable them though, I didn't check.

`mkdir -p ~/.themes/THEMENAME/gnome-shell/`
```
cat << EOF > ~/.themes/THEMENAME/gnome-shell/gnome-shell.css
.dash-label {
  width: 0px;
  height: 0px;
  padding: 0px;
  border: none;
}
EOF
```

I tried better CSS rules like "display none" or "visibility hidden" but it doesn't work.

**Comment by nilsboy** [comment]
It is really annoying that the tooltips stay visible after an app button is clicked. I usually have my terminal set to maximized and when I switch to it using the mouse I leave the mouse where it is and start typing and the tooltip is in the way of my terminal window.

Thanx for considering this.

**Comment by nilsboy** [comment]
Quick hack to remove the tooltips in Ubuntu 20.04:

```sh
perl -i -pe 's/(function itemShowLabel\(\)\s*\{)/$1\nreturn\n/gs' \
  ~/.local/share/gnome-shell/extensions/dash-to-dock@micxgx.gmail.com/appIcons.js
```
